StepDefinitions/Feature1_LanguageStepDefinitions.cs

```diff
@@ -0,0 +1,242 @@
+using Mars_SpecFlowProject1.Pages;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using TechTalk.SpecFlow;
+using Mars_SpecFlowProject1.Utils;
+using NUnit.Framework;
+using SpecFlowProject_MarsProject.Utils;
+using OnboardTaskProjectMars.Pages;
+
+namespace ProjecrMarsOnboardingtask.StepDefinitions
+{
+    [Binding, Scope(Feature = "Feature1_Language")]
+    public class Feature1_LanguageStepDefinitions : HookBase
+    {
+        private readonly LoginPage loginPageObj;
+        private readonly AddLanguageTab languageTabObj;
+        private readonly EditLanguage editLanguageObj;
+        private readonly DeleteLanguage deleteLanguageObj;
+        private readonly AddLanguagewithEmptyName addLanguagewithEmptyNameObj;
+        private readonly AddlanguagewithDuplicateEntry addLanguagewithDuplicateEntryObj;
+        private readonly AddMultipleLanguageRecord AddMultipleLanguageRecordObj;
+        private readonly EditLanguageWithEmptyName editLanguageWithEmptyNameObj;
+        private readonly EditLanguageWithDuplicateEntry editLanguageWithDuplicateEntryObj;
+        private readonly CleanLanguageData cleanLanguageDataObj;
+        public Feature1_LanguageStepDefinitions()
+        {
+            loginPageObj = new LoginPage();
+            languageTabObj = new AddLanguageTab();
+            editLanguageObj = new EditLanguage();
+            deleteLanguageObj = new DeleteLanguage();
+            addLanguagewithEmptyNameObj = new AddLanguagewithEmptyName();
+            addLanguagewithDuplicateEntryObj = new AddlanguagewithDuplicateEntry();
+            AddMultipleLanguageRecordObj = new AddMultipleLanguageRecord();
+            editLanguageWithEmptyNameObj = new EditLanguageWithEmptyName();
+            editLanguageWithDuplicateEntryObj = new EditLanguageWithDuplicateEntry();
+            cleanLanguageDataObj = new CleanLanguageData();
+
+        }
+       
+        [Given(@"User Logs into Mars portal and navigates to language tab")]
+        public void GivenUserLogsIntoMarsPortalAndNavigatesToLanguageTab()
+        {
+            loginPageObj.login(driver);
+        }
+        // Clean data
+        [When(@"User deletes All languages record")]
+        public void WhenUserDeletesAllLanguagesRecord()
+        {
+            cleanLanguageDataObj.Deletelanguage(driver);
```

Comment:
Please do not pass the driver In action methods (methods which in pages doing things like create, delete etc ) instead get the pages to extend to common driver class